src/ConnectedMode/ConnectedMode.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <!-- NOTE: this assembly should NOT have any dependencies on Visual Studio assemblies -->
```

Comment:
This only really applies to `Core` and `Roslyn.Suppressions`. It doesn't have to be the case for this assembly.